notebooks/203-meter-reader/203-meter-reader.ipynb

```diff
@@ -0,0 +1,730 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Industrial Meter Reader\n",
+    "This notebook shows how to create a industrial meter reader with OpenVINO Runtime. We use the PaddlePaddle pre-trained model [PPYOLOv2](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.4/configs/ppyolo) and [DeepLabV3P](https://github.com/PaddlePaddle/PaddleSeg/tree/release/2.5/configs/deeplabv3p) to built-up a multiple inference task pipeline:\n",
+    "\n",
+    "1) Run detection model to find out the meters, and then crop them from the orgin photo.\n",
+    "2) Run segmentation model on these croped meters to get the pointer and scale instance.\n",
+    "3) Find the location of the poiner in scale map.\n",
+    "\n",
+    "<img align='center' src=\"https://user-images.githubusercontent.com/91237924/166137115-67284fa5-f703-4468-98f4-c43d2c584763.png\" alt=\"Task Pipeline\" width=\"1000\"/>\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import os.path as osp\n",
+    "import numpy as np\n",
+    "import math\n",
+    "import cv2\n",
+    "import tarfile\n",
+    "import matplotlib.pyplot as plt\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "from notebook_utils import download_file, segmentation_map_to_image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Model and Test Image\n",
+    "Download PPYolov2 and DeepLabV3P pre-trained models from PaddlePaddle community."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_DIR = \"model\"\n",
+    "DATA_DIR = \"data\"\n",
+    "DET_MODEL_LINK = \"https://bj.bcebos.com/paddlex/examples2/meter_reader/meter_det_model.tar.gz\"\n",
+    "SEG_MODEL_LINK = \"https://bj.bcebos.com/paddlex/examples2/meter_reader/meter_seg_model.tar.gz\"\n",
+    "DET_FILE_NAME = DET_MODEL_LINK.split(\"/\")[-1]\n",
+    "SEG_FILE_NAME = SEG_MODEL_LINK.split(\"/\")[-1]\n",
+    "IMG_LINK = \"https://user-images.githubusercontent.com/91237924/170696219-f68699c6-1e82-46bf-aaed-8e2fc3fa5f7b.jpg\"\n",
+    "IMG_FILE_NAME = IMG_LINK.split(\"/\")[-1]\n",
+    "\n",
+    "os.makedirs(MODEL_DIR, exist_ok=True)\n",
+    "\n",
+    "download_file(DET_MODEL_LINK, directory=MODEL_DIR, show_progress=True)\n",
```

Comment:
Downloading these models is extremely slow. Is it possible to speed up it? Some other server?

Reply:
Sorry for that, since this server is hosted by Baidu, it's fast for PRC region.